functional-tests/src/test/java/com/redhat/che/functional/tests/TestTestClass.java

```diff
@@ -25,18 +46,62 @@
 
   private static final Logger LOG = LoggerFactory.getLogger(TestTestClass.class);
 
-  @Inject private TestWorkspace testWorkspace;
-  @Inject private ProjectExplorer projectExplorer;
-  @Inject private Menu menu;
+//  @Inject private TestWorkspace testWorkspace;
```

Comment:
IMO we should remove all commented code

Reply:
yes, I will clean it up when I squash the commits.